forge-gui/res/cardsfolder/upcoming/sadistic_shell_game.txt

```diff
@@ -0,0 +1,8 @@
+Name:Sadistic Shell Game
+ManaCost:4 B
+Types:Sorcery
+A:SP$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBChooseCard | SubAbility$ DBDestroy | InTurnOrder$ True | StackDescription$ SpellDescription | SpellDescription$ Starting with the next opponent in turn order, each player chooses a creature you don’t control. Destroy the chosen creatures.
+SVar:DBChooseCard:DB$ ChooseCard | Choices$ Creature | Reveal$ True | Defined$ Player.IsRemembered | ControlledByPlayer$ Player.Opponent | Mandatory$ True | ChoiceTitle$ Choose a creature to destroy | RememberChosen$ True | AILogic$ OppPreferred
+SVar:DBDestroy:DB$ DestroyAll | ValidCards$ Card.Creature+IsRemembered | SubAbility$ DBCleanup | StackDescription$ Destroy the chosen creatures.
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True | ClearRemembered$ True
+Oracle: Starting with the next opponent in turn order, each player chooses a creature you don’t control. Destroy the chosen creatures.
```

Comment:
spacing

Reply:
Addressed in recent [commit ](https://github.com/Card-Forge/forge/pull/6226/commits/261308bf219e657dff36908625ce7760906fa91a)